docs/source/api.rst

```diff
@@ -0,0 +1,11 @@
+pyspool API Documentation
+=========================
+
+Spool
+-----
+.. automodule:: spool
+
+.. autoclass:: Spool
+    :members:
+
+.. :members: __init__, register_piece, register, editions, consigned_registration, transfer, consign, unconsign, loan, migrate, refill_main_wallet, refill, simple_spool_transaction, select_inputs
```

Comment:
I assume this is where the current [documentation section](https://github.com/ascribe/pyspool#documentation) in the README will go?